### PR TITLE
Update Ports in Install Guide

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/9598

### CODE REVIEW

**Comment by Loquacity** [comment]
Ready for review :)

File: modules/installation/pages/ports.adoc [hunk 1]
```diff
@@ -1,210 +1,101 @@
...
+== External Proxy Ports
+
+[cols="1,1,1,1,1", options="header"]
+.External Port Requirements for {productname} Proxy
+|===
+| Port number | Direction | Protocol | Used By | Notes
+| 22          | Inbound   |          | | Required for ssh-push and ssh-push-tunnel contact methods.
+Clients connected to the proxy initiate check in on the server and hop through to clients.
+| 67          | Inbound   | TCP/UDP  | DHCP    | Required only if clients are requesting IP addresses from the server.
+| 69          | Inbound   | TCP/UDP  | TFTP | Required if server is used as a PXE server for automated client installation.
```
Comment (keichwa) [inline modules/installation/pages/ports.adoc]:
```suggestion
| 69          | Inbound   | TCP/UDP  | TFTP | Required if the server is used as a PXE server for automated client installation.
```

File: modules/installation/pages/ports.adoc [hunk 2]
```diff
@@ -1,210 +1,101 @@
...
-+
-
-To reach the {productname} server or {susemgrproxy} server.
 
-5222::
-Outbound / TCP
-+
 
-For push actions with the server or proxy server.
+== External Server Ports
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/installation/pages/ports.adoc]: OK, so, I agree with you 100%. *However*, I would rather get this merged as it is (assuming it's technically correct), and then tackle the restructure in a new issue. I have other things that are more important on my time right now.

[the remark this answers is not in the document]
Comment (moio) [reply modules/installation/pages/ports.adoc]: Then for now at least let's make clear what those refer to, adding definitions for:

External inbound port: a port to be opened on the {prouctname} Server or Proxy
External outbound port: a destination port that the {prouctname} Server or Proxy might use to open connections
Internal port: a port only used internally, exclusively accessible from localhost

**Comment by Loquacity** [comment]
@moio can you please review so we can merge this one? I'll address the outstanding comments in a future PR.

File: modules/installation/pages/ports.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (moio) [reply modules/installation/pages/ports.adoc]: Please add the definitions above and then this PR can be merged FMPOV

[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/installation/pages/ports.adoc]: Oops, sorry, looks like I forgot to push.

**Pull request merged**